frontend/src/pages/ErrorsV2/ErrorInstance/ErrorInstance.tsx

```diff
@@ -345,14 +337,23 @@ const ErrorInstance: React.FC<Props> = ({ errorGroup }) => {
 								}
 								const query = stringifyLogsQuery(queryParams)
 								const logCursor = errorObject.log_cursor
+								const errorTs = moment(errorObject.timestamp)
+								const params = createSearchParams({
+									query,
+									start_date: errorTs
+										.add(-5, 'minutes')
+										.toISOString(),
+									end_date: errorTs
+										.add(5, 'minutes')
```

Comment:
Pretty sure this isn't actually setting 5 minutes ahead but just resetting back to `errorObject.timestamp` given that mutable objects aren't immutable.

```js
const now = new Date()
console.log(now)
=> Wed Mar 22 2023 09:44:21 GMT-0600 (Mountain Daylight Time)

const m = moment(now)
m.toDate()
=> Wed Mar 22 2023 09:44:21 GMT-0600 (Mountain Daylight Time)

const startDate = m.add(-5, 'minutes').toDate()
console.log(startDate)
=> Wed Mar 22 2023 09:39:21 GMT-0600 (Mountain Daylight Time)

const endDate = m.add(5, 'minutes').toDate()
console.log(endDate)
=> Wed Mar 22 2023 09:44:21 GMT-0600 (Mountain Daylight Time)
```

Creating two moment objects is probably the easiest way to get around this annoying quirk of moment.

Reply:
good catch!